src/adapters/postgres/user-adapter.ts

```
@@ -2966,6 +2969,263 @@ export class PostgresUserService implements IServicelocator {
+      };
+    }
+
+    const anonymizedIdentity = this.generateAnonymizedIdentity(user.userId);
+
+    // 1. Keycloak: only email/username/firstName/lastName. Account stays enabled, no session invalidation.
+    const keycloakResult = await updateUserInKeyCloak(
+      {
+        userId: user.userId,
+        email: anonymizedIdentity,
+        username: anonymizedIdentity,
+        firstName: 'deleted',
```

ESLint

[error] Replace `'deleted'` with `"deleted"`

(prettier/prettier)